modules/openy_features/openy_media/modules/openy_media_image/openy_media_image.info.yml

```diff
@@ -15,4 +15,5 @@ dependencies:
   - openy_media
   - user
   - views
+  - views_infinite_scroll
```

Comment:
Since pager has been reverted back to mini pager, we no longer need this in dependencies.